mne/gui/_core.py

```diff
@@ -0,0 +1,483 @@
+# -*- coding: utf-8 -*-
+"""Shared GUI classes and functions."""
```

Comment:
I did not look at this file much at all because I assume it is 99% copy-paste from the GUI code in `main`. Is that correct? If not, can you point out which bits of the code are new in this file that I should focus on?

Reply:
Yes that is correct, I just tried to copy-paste and move all the code that is specific to slice browsing to the `core.py` file with a few small changes to make it work.